src/tests/mocks/accesscontrol_mocks.cairo

```diff
@@ -148,37 +148,41 @@ mod SnakeAccessControlPanicMock {
     #[storage]
     struct Storage {}
 
-    #[external(v0)]
-    fn has_role(self: @ContractState, role: felt252, account: ContractAddress) -> bool {
-        panic!("Some error");
-        false
-    }
-
-    #[external(v0)]
-    fn get_role_admin(self: @ContractState, role: felt252) -> felt252 {
-        panic!("Some error");
-        3
-    }
-
-    #[external(v0)]
-    fn grant_role(ref self: ContractState, role: felt252, account: ContractAddress) {
-        panic!("Some error");
-    }
-
-    #[external(v0)]
-    fn revoke_role(ref self: ContractState, role: felt252, account: ContractAddress) {
-        panic!("Some error");
-    }
-
-    #[external(v0)]
-    fn renounce_role(ref self: ContractState, role: felt252, account: ContractAddress) {
-        panic!("Some error");
-    }
-
-    #[external(v0)]
-    fn supports_interface(self: @ContractState, interface_id: felt252) -> bool {
-        panic!("Some error");
-        false
+    #[abi(per_item)]
+    #[generate_trait]
+    impl ExternalImpl of ExternalTrait {
```

Comment:
i thought the impl would save us the external macros, but now i'm not sure it's needed especially if we don't use the generated trait. either way is fine.

Reply:
I'll remove the `per_item` from all the mocks